packages/marko-web/components/page/metadata/components/common.marko

```diff
@@ -28,6 +29,10 @@ $ const image = imageSrc || fallbackImage;
 <if(image)>
   <!-- Image (standard, OpenGrah, and Schema.org respectively) -->
   <meta name="image" property="og:image" item-prop="image" content=image />
+  <if(imageDimensions && imageDimensions.width && imageDimensions.height)>
+    <meta property="og:image:width" content=imageDimensions.width />
```

Comment:
The width and height values should also be stringified, so:
```marko
content=`${imageDimensions.height}`
```

Once complete can you also update the screenshot in the PR? Thanks!